tensorboard/plugins/core/core_plugin.py

```diff
@@ -285,6 +292,31 @@ def _serve_experiment_runs(self, request):
         results = []
         return http_util.Respond(request, results, "application/json")
 
+    @wrappers.Request.application
+    def _serve_notifications(self, request):
+        """Serve a JSON runs of an experiment, specified with query param
```

Comment:
Technically google python style is to have the "summary line" (first line of docstring) never exceed 1 line:
https://google.github.io/styleguide/pyguide.html#381-docstrings

In this case it also needs to be updated since I think this is from `experiment_runs` :)  I'd suggest something like `"""Serve JSON payload of notifications to show in the UI."""` or similar.